### ISSUE TITLE
Docker version doesn't run; segmentation fault or std::bad_alloc

### ISSUE THREAD

**Issue by 6r1d** [OPHoperHPO/image-background-remove-tool#75]
Hello, I've tried to install CarveKit recently and saw some problems with it, so I'm going to record them.

When I run the Docker version, what I see is either:

```
carvekit_api_1  | terminate called after throwing an instance of 'std::bad_alloc'
carvekit_api_1  |   what():  std::bad_alloc
carvekit_api_1  | Aborted
```

or

```
carvekit_api_1  | Segmentation fault
```

My machine specs are:

* Linux 5.17.8 (Artix)
* Docker version 20.10.16
* docker-compose version 1.29.2
* AMD Ryzen 5 4500U CPU
* 16 Gb RAM

If I replace the command with `yes`, I see the container running, so for now I can only point out that the server breaks because of one of the libraries, the server code itself is very simple and shouldn't cause that error.

I must also note that it repeats with all three `CARVEKIT_SEGMENTATION_NETWORK` values, be it `u2net`, `basnet`, `deeplabv3`.

**Comment by OPHoperHPO** [comment]
Hey,

What version of the docker image are you building? CPU or CUDA?
Can you enter a container shell and run python3? Then try to import `torch`, `numpy`,`cv2`, `torchvision`, `pillow`, `uvicorn`, `fastapi` separately.
What version of python are you see?

I tried to reproduce this error in my docker container and everything works as expected.

My test machine specs:
```
Linux kernel: 5.17.1-3
Docker version 20.10.12
docker-compose version 2.4.1
Intel-based cpu
python 3.8.2
```

**Comment by 6r1d** [comment]
> If you changed Dockerfile or docker-compose.yml, please upload it. It is impossible to debug issue in an unknown enviroment.

The small change that was left in CarveKit is me replacing the server launch with `yes` so I can do `docker exec` after segfault.

```bash
docker exec -ti image-background-remove-tool_carvekit_api_1 /bin/bash
uvicorn carvekit.web.app:app --host 0.0.0.0 --port 5000
```

I'm building a CPU version of the tool. But it just got weirder: I did `docker container prune` and `docker image prune` and after, CarveKit web interface ran normally. It successfully removes the backgrounds now.

I removed the "yes" part after, so I guess it was some damaged file breaking CarveKit. Thanks, I am closing this issue.
